request-interfaces.go

```diff
@@ -149,6 +149,11 @@ type ListerAt interface {
 	ListAt([]os.FileInfo, int64) (int, error)
 }
 
+// CloserListerAt is a ListerAt that implements the Close method.
+type CloserListerAt interface {
```

Comment:
Hm, convention seems to suggest this would be `ListAtCloser`.

Also, ¿typo? the interface doesn’t include `ListAt(…)` method?

Reply:
ops, fixed